README.md

```diff
@@ -2,8 +2,34 @@ For some context on why we think having a code of conduct for our organization i
 
 # DigitalOcean's Engineering Code of Conduct
 
+At DigitalOcean we strive to be a values driven company.
```

Comment:
Grammarly suggestion: "values driven" to "values-driven".

Reply:
Agree. fixing that